mappings/net/minecraft/world/gen/chunk/ChunkGeneratorType.mapping

```diff
@@ -2,16 +2,31 @@ CLASS net/minecraft/class_5284 net/minecraft/world/gen/chunk/ChunkGeneratorType
 	FIELD field_24514 defaultBlock Lnet/minecraft/class_2680;
 	FIELD field_24515 defaultFluid Lnet/minecraft/class_2680;
 	FIELD field_24516 config Lnet/minecraft/class_5311;
+	FIELD field_24780 OBJECT_CODEC Lcom/mojang/serialization/Codec;
```

Comment:
the codec for plain chunk generator type, without string shortcuts? imo the one that does not accept shortcuts should be `CODEC`, and one that accept alternatives as `PRESET_CODEC` etc

Reply:
Should I also rename `ChunkGeneratorType.Preset.PRESET_CODEC` to `CODEC` instead?

Reply:
wait, so what does this codec do 🤔

Reply:
The preset's own codec is the only codec so it can be named `CODEC`

Reply:
Yeah, I hadn't noticed they weren't strictly in the same class yet